crates/bitwarden-core/src/client/internal.rs

```diff
@@ -304,6 +304,12 @@ impl InternalClient {
         user_key: SymmetricCryptoKey,
         key_state: UserKeyState,
     ) -> Result<(), EncryptionSettingsError> {
+        let span = tracing::info_span!(
+            "User Crypto Initialization",
+            user_id = ?self.get_user_id(),
+        );
+        let _enter = span.enter();
```

Comment:
You can also use the `entered` function, which takes ownership of the span and doesn't require two variable definitions:
```rust
        let _span_guard = tracing::info_span!(
            "User Crypto Initialization",
            user_id = ?self.get_user_id(),
        )
        .entered();
```

https://docs.rs/tracing/latest/tracing/struct.Span.html#method.entered